### DISCUSSION TITLE
修复 MacVim 9.0 的 Python3 支持 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#262] [Announcements]
# 修复 MacVim 9.0 的 Python3 支持 — 码志

前两天刚刚升级到了 MacVim 9.0 的最新版本，日常编辑编辑文字没遇到过什么问题，直到今天动了一下插件。

https://mazhuang.org/2022/09/22/enable-python3-in-macvim/

**Comment by wuxiaobai137** [comment]
hello
